ibm_i/datadog_checks/ibm_i/config_models/validators.py

```diff
@@ -0,0 +1,15 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+
+def instance_severity_threshold(value, *, field):
+    v = int(value)
+    if v < 0 or v > 99:
+        raise ValueError("severity threshold must be in the range [0,99]")
+    return v
+
+
+def instance_query_timeout(value, *, field):
+    v = int(value)
+    return v
```

Comment:
I think we want the same kind of validator for the other two timeout values (`job_query_timeout` and `system_mq_query_timeout`)

Reply:
I used [OpenAPI's minimum and maximum](https://swagger.io/docs/specification/data-models/data-types/) for this since you get an automatic validator that way 😄 (see 9139eac4339cc5744c810bf5f60b30186b589126)